rootio/wbuffer.go

```diff
@@ -153,19 +154,12 @@ func (w *WBuffer) WriteString(v string) { //FIXME :works only on small strings
 	if l < 255 {
 		b[0] = byte(l)
 		copy(b[1:], []byte(v))
-		var n int
-		n, w.err = w.w.Write(b)
-		w.w.c += n
+		w.w.Write(b)
```

Comment:
here you drop the error that is returned by `w.w.Write` on the floor.
you should store it inside `w.err`.